demos/browser/README.md

```diff
@@ -44,6 +44,7 @@ For messaging session, make sure your role policy contains `chime:Connect` and `
 1. Navigate to the `demos/browser` folder: `cd demos/browser`
 
 2. Start the demo application in 'hot reload' mode: `npm run start:hot`
+Note: The first time when the demo is run there might be a dependency error. Please run `npm run start` first as that will install the required dependencies. After that, `npm run start:hot` should not throw any errors.
```

Comment:
If preferred, I can change the first line to "The first time when the repository is cloned the user might get a dependency error"? The current language might be too broad as the user might confuse any dependency error with this scenario.

Reply:
How about if we say:
2. Start the demo application: `npm run start`
> Once you run the application for the first time, you can then use `npm run start:hot` to run the application in 'hot reload' mode.

Reply:
For the note we have used a forward arrow ">" <space> <your sentence>. (The markdown quote's syntax)
You may already know, but for more info you can check this GitHub guide:
https://guides.github.com/features/mastering-markdown/

Reply:
Done, thank you.